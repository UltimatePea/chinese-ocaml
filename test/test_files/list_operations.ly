// 骆言测试程序：列表操作
// Luoyan Test Program: List Operations

让 列表 = [1, 2, 3, 4, 5]

让 求和 = 函数 lst ->
  匹配 lst 与
  | [] -> 0
  | [head, ...tail] -> head + 求和 tail

让 结果 = 求和 列表
打印 "列表求和: "
打印 结果