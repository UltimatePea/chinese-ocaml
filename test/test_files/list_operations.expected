列表求和: 
15